Need to support cobra 1.6

Cobra 1.6 has modified its default help template.  This breaks cobrautil which needs to manipulate the original string.
This has come up when trying to upgrade to Cobra 1.6 in Tanzu Framework where we get the error:
```
panic: Expected to find available commands section in spf13/cobra default usage template
```
The problematic code is: https://github.com/cppforlife/cobrautil/blob/d60711905d65368620738bff69dcd0ee84c3f69c/help_sections.go#L36-L43

The change to Cobra can be seen here: https://github.com/spf13/cobra/commit/2169adb5749372c64cdd303864ae8a444da6350f#diff-4d7d1923693fc5ce892add2ea2907a744e77ea0b50c1939ccc5067cb48a466a3L514
and was part of https://github.com/spf13/cobra/pull/1003

I'll post a PR

--- a/help_sections.go
+++ b/help_sections.go
@@ -30,17 +30,35 @@ type HelpSection struct {
 }
 
 func HelpSectionsUsageTemplate(sections []HelpSection) string {
-	unmodifiedCmd := &cobra.Command{}
-	usageTemplate := unmodifiedCmd.UsageTemplate()
+	usageTemplate := `Usage:{{if .Runnable}}
+  {{.UseLine}}{{end}}{{if .HasAvailableSubCommands}}
+  {{.CommandPath}} [command]{{end}}{{if gt (len .Aliases) 0}}
+
+Aliases:
+  {{.NameAndAliases}}{{end}}{{if .HasExample}}
+
+Examples:
+{{.Example}}{{end}}{{if .HasAvailableSubCommands}}
+
+Available Commands:{{range .Commands}}{{if (or .IsAvailableCommand (eq .Name "help"))}}
+  {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}{{end}}{{if .HasAvailableLocalFlags}}
+
+Flags:
+{{.LocalFlags.FlagUsages | trimTrailingWhitespaces}}{{end}}{{if .HasAvailableInheritedFlags}}
+
+Global Flags:
+{{.InheritedFlags.FlagUsages | trimTrailingWhitespaces}}{{end}}{{if .HasHelpSubCommands}}
+
+Additional help topics:{{range .Commands}}{{if .IsAdditionalHelpTopicCommand}}
+  {{rpad .CommandPath .CommandPathPadding}} {{.Short}}{{end}}{{end}}{{end}}{{if .HasAvailableSubCommands}}
+
+Use "{{.CommandPath}} [command] --help" for more information about a command.{{end}}
+`
 
 	const defaultTpl = `{{if .HasAvailableSubCommands}}
 
 Available Commands:{{range .Commands}}{{if (or .IsAvailableCommand (eq .Name "help"))}}
   {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}{{end}}`
-
-	if !strings.Contains(usageTemplate, defaultTpl) {
-		panic("Expected to find available commands section in spf13/cobra default usage template")
-	}
 
 	newTpl := "{{if .HasAvailableSubCommands}}"
 

